File: GotadeOuroDefault.Web/Modules/ServerTypes/Cliente/ClienteForm.ts
import { StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface ClienteForm {
    Nome: StringEditor;
    Rg: StringEditor;
    Cpf: StringEditor;
    Cnh: StringEditor;
    OrgaoEmissor: StringEditor;
    Telefone: StringEditor;
    Estado: StringEditor;
}

export class ClienteForm extends PrefixedContext {
    static formKey = 'Cliente.Cliente';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ClienteForm.init)  {
            ClienteForm.init = true;

            var w0 = StringEditor;

            initFormType(ClienteForm, [
                'Nome', w0,
                'Rg', w0,
                'Cpf', w0,
                'Cnh', w0,
                'OrgaoEmissor', w0,
                'Telefone', w0,
                'Estado', w0
            ]);
        }
    }
}

File: GotadeOuroDefault.Web/Modules/Cliente/Cliente/ClienteForm.cs
using Serenity.ComponentModel;

namespace GotadeOuroDefault.Cliente.Forms;

[FormScript("Cliente.Cliente")]
[BasedOnRow(typeof(ClienteRow), CheckNames = true)]
public class ClienteForm
{
    public string Nome { get; set; }
    public string Rg { get; set; }
    public string Cpf { get; set; }
    public string Cnh { get; set; }
    public string OrgaoEmissor { get; set; }
    public string Telefone { get; set; }
    public string Estado { get; set; }
}